image/image.go

```diff
@@ -217,8 +293,6 @@ func bootstrapToRootDir(sto Store, model *asserts.Model, opts *Options) error {
 		Channel:   opts.Channel,
 	}
 
-	// FIXME: support sideloading snaps by copying the boostrap.snaps
-	//        first and keeping track of the already downloaded names
 	snaps := []string{}
 	snaps = append(snaps, opts.Snaps...)
```

Comment:
A comment here maybe that the order of the append is important? I.e. that opts.Snaps needs to be first so that side-loading of snaps like ubuntu-core work?